### ISSUE TITLE
Losing Internet mid Export Template download crashes Godot

### ISSUE THREAD

**Issue by SoloRobo** [godotengine/godot#56057]
### Godot version

3.4.1

### System information

Windows 10

### Issue description

If you "turn off" the Internet whilst the "Export Template Manager" is downloading it crashes Godot

### Steps to reproduce

Open Export Template Manager
Click "Download and Install"
"Turn off" Internet, unknown if turning on Aeroplane mode replicates the issue, or simply unplugging ethernet/turning off wifi router

**Comment by Calinou** [comment]
> Turning Windows 10 Aeroplane/Flight mode is recognised by Godot as something to manage.

Does this mean Godot doesn't crash if you enable airplane mode while downloading export templates?

**Comment by SoloRobo** [comment]
> Does this mean Godot doesn't crash if you enable airplane mode while downloading export templates?

Correct, I tested this and Godot appears to timeout in the background then display a message that it will download in the background.

However I just tested this again, i.e. put Windows 10 in "Flight Mode", in the hope of getting a screenshot and it was unable to continue the download, I went to cancel as the message was not appearing and Godot went through a "Not Responding" phase, recovered and showed the message (after it had frozen):

![Capture](https://user-images.githubusercontent.com/20973406/146668648-cc596ab3-f421-4dae-b75e-c25e4ca8d1cd.PNG)
